### DISCUSSION TITLE
paste adt.lic at lib folder, compile and crash after app lunch

### DISCUSSION THREAD

**Discussion by Lantui** [airsdk/Adobe-Runtime-Support#625] [Q&A]
## Problem Description
just purchase the adobe license and they give adt.lic file to paste on AIR SDK 33.1.1.259 "lib" folder, lunch the apps and it crash. 
i remove adt.lic file, it work fine on IOS.

using AIR SDK 33.1.1.259
IOS only it crash android working fine.
using Adobe Animate CC 2020
No starling

**Comment by ajwfrost** [comment]
That's the opposite way round to normal! Are you able to email us your IPA file when built with the adt.lic file present? Andrew.Frost at Harman.com is simplest - let me know if you need a link from us for a file upload location..

thanks

**Comment by Lantui** [comment]
Already send the IPA to you

**Comment by xs2bas** [comment]
Hi Andrew, I didn't want to open a whole new issue for my question which is a little off topic, but my license file isn't working anymore giving this warning:

WARNING: ADT license file for developer ID "......." reported as: INVALID
License file has been removed: please contact adobe.support@harman.com to renew your license
License file is present but not valid

... I received the file in mid July so it should still work.
Ik contacted through mail, but I cannot wait another 10 working days for the solution, so I hope you can assist this way :)

**Comment by ReginaSun** [comment]
Hi, I got the same problem.

When the device is Y12(1904) of VIVO and the OS is android11, I lunch my app then it crash.

But if I remove adt.lic, it works successfully.

Do you guys have any idea for this?

AIR SDK：33.1.1.345

Thanks a lot.

**Reply by ajwfrost** [reply]
Hi - could you get a logcat output for this please? This would hopefully give us the necessary information to check why it's crashing..
thanks
